pkg/clockcache/cache_bench_test.go

```diff
@@ -233,3 +235,8 @@ func BenchmarkInsertConcurrent(b *testing.B) {
 		})
 	}
 }
+
+func BenchmarkMemoryEmptyCache(b *testing.B) {
+	b.ReportAllocs()
+	WithMax(uint64(b.N))
+}
```

Comment:
Please paste the benchmark results as a comment to the PR.

Reply:
It was 100-120 bytes per element. Which is where I got the 120 for the estimate